lib/mssql.js

```diff
@@ -0,0 +1,215 @@
+import Ajv from "ajv";
+import JSONStream from "JSONStream";
+import {json} from "micro";
+import mssql from "mssql";
+import {notFound, failedCheck, badRequest} from "./errors.js";
+
+const TYPES = mssql.TYPES;
+
+const pools = new Map();
+
+// TODO: Confirm these are the right SQL verbs.
+const READ_ONLY = new Set(["SELECT", "SHOW DATABASES", "SHOW VIEW", "USAGE"]);
+
+const ajv = new Ajv();
+const validate = ajv.compile({
+  type: "object",
+  additionalProperties: false,
+  required: ["sql"],
+  properties: {
+    sql: {type: "string", minLength: 1},
+    params: {type: "array"},
+  },
+});
+
+// Since we want to work with more than one DB,
```

Comment:
I'm a little confused by this comment and what we mean by a custom version? Maybe we don't need this comment?